README.md

```diff
@@ -77,6 +77,45 @@ header with proper audio byte length into the beginning of the destination
 (though if your destination _is_ a regular file, you should use the the
 `FileWriter` class instead).
 
-### FileWriter()
+Default options: 
 
-The `FileWriter` class.
+```JSON
+{
+  "channels": 2,
+  "sampleRate": 44100,
+  "bitDepth": 16
+}
+```  
+
+
+### FileWriter(path, options)
+
+The `FileWriter` class is, essentially, a combination of `fs.createWriteStream()` and the above `Writer()` class, except it automatically corrects the header after the file is written. Options are passed to both `Writer()` and `fs.createWriteStream()`.
+
+Example usage with `mic`:
+
+```js
+var FileWriter = require('wav').FileWriter;
+var mic = require('mic'); // requires arecord or sox, see https://www.npmjs.com/package/mic
```

Comment:
[`line-in`](https://www.npmjs.com/package/line-in) doesn't depend on anything being installed 😎 

*disclaimer: it's my package*